core/client/app/components/gh-autonav-toggle.js

```diff
@@ -1,13 +0,0 @@
-import Ember from 'ember';
-
-export default Ember.Component.extend({
-    classNames: ['gh-autonav-toggle'],
-
-    maximise: false,
-
-    click: function () {
-        this.toggleProperty('maximise');
-
-        this.sendAction('onClick');
-    }
-});
```

Comment:
Boo. I did a `git mv` on this, but also edited it and now it's all in one commit.... should i pull it apart into two steps?